lib/form-validation.js

```diff
@@ -6361,6 +6366,24 @@
 
     return;
   };
+
+  var validationJsNumericality = validate.validators.numericality.bind(validate);
+  validate.validators.numericality = function (value, options, attribute, attributes, globalOptions) {
+    const delocalisedValue = value != undefined ? value.replace(",", ".").replace(" ", "") : value;
```

Comment:
Ik vermoed dat hier dan is waar "men" (AIV) zegt "alle getallen moeten ingegeven worden met , als decimal separator en ' ' als thousand separator?
Moet er ook niks zijn met hoeveel cijfers na de komma er maximaal kunnen ingegeven worden? Of zit dat bij pattern dan?

Reply:
Het is inderdaad mogelijk om dat in de pattern in te stellen.

Reply:
Dat laatste houdt trouwens wel in dat de afnemers hier rekening mee moeten houden als ze de waarde van de input uitlezen. Want parseInt/parseFloat werkt niet (correct) als er spaties of komma's in zitten...

Reply:
In combinatie met vl-ui-pattern zou dan de `getRawValue` gebruikt kunnen worden.

Reply:
In de afnemers van vl-input, ja, maar ook in form-validation zelf (dus hier, op de plaats in de code waar deze comment op staat)? 
Want het is niet gezegd dat form-validation _steeds_ tesamen met de vl-pattern gebruikt wordt, natuurlijk.

Reply:
Indien dat het geval is, zou ik ook in de vl-form-validation een `getRawValue` introduceren eerder dan applicatief de value te manipuleren. Stel dat morgen het formaat centraal wijzigt, willen we geen applicatie code moeten aanpassen.

Reply:
In ons geval zullen ze er wel steeds beide staan dus hebben we dat nog niet nodig. En zou ik het dus ook niet meer meenemen nu als we de getRawValue al op pattern hebben.

Reply:
vl-form-validation zelf hoeft geen `getRawValue` te exposen volgens mij, want vl-form-validation verknoeit het formaat niet. Enkel de vl-pattern knoeit met het formaat en heeft dus een `getRawValue` nodig.
Ik heb het hier over de replace's die we nu in vl-form-validation doen: `value.replace(",", ".").replace(" ", "")`

Reply:
Dan begrijp ik niet helemaal waarom afnemers rekening moeten houden met de interne code (value manipulatie) van vl-form-validation?

Reply:
Na de demo nog eens te bekijken ben ik mee, maar lijkt het me geen probleem te zijn. Indien afnemers toelaten dat er een komma en spatie gebruikt mogen worden en bijgevolg niet de vl-pattern gebruiken, moeten ze die value applicatief inderdaad correct opvangen.

Reply:
Dat moeten ze ook niet (rekening houden met value manipulatie van **vl-form-validation**). 
Deze comment heeft het over het feit dat de vl-pattern values manipuleert en dat we als gebruiker van de input's value (vl-form-validation gebruikt deze, om te valideren) er rekening mee moeten houden. Hence: de replaces die we moeten doen.

Reply:
Mijn vraag was dan vooral: moet ik hier in vl-form-validation gewoon de replaces voor de komma's en spaties laten staan as-is?

Reply:
We zouden in de vl-form-validation code ook de `getRawValue` kunnen gebruiken zodat die manipulatie niet op 2 plaatsen zit, maar aangezien de `getRawValue` voorlopig alleen onderdeel is van onze eigen webcomponent en dus centraal nooit gekend zal zijn, zou ik het voorlopig zo houden. Al neem ik het zeker mee bij de centrale communicatie.